src/runtime_src/core/tools/xbtracer/src/lib/detail/windows/logger.h

```diff
@@ -0,0 +1,88 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+
+namespace xrt::tools::xbtracer {
+
+constexpr std::string_view path_separator = "\\";
+
+/*
+ * Wrapper api for time formating
+ * */
+inline std::tm localtime_xp(std::time_t timer)
+{
+  std::tm bt{};
+  localtime_s(&bt, &timer);
+  return bt;
+}
+
+std::string get_env(std::string key)
```

Comment:
It's not for free to pass std::string by value; use const reference?

Reply:
Done